WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsHostingController.swift

```diff
@@ -1,12 +1,15 @@
 import SwiftUI
 
 final class PaymentMethodsHostingController: UIHostingController<HostedPaymentMethodsView> {
+    weak var parentController: UIViewController?
```

Comment:
It may be good adding some comments about why we need this property!

Reply:
Comment added as suggested. Thanks!